### PR TITLE
[11.0][mig] sale_fixed_discount

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration.

Introduces some changes to improve the compatibility with account_invoice_fixed_discount https://github.com/OCA/account-invoicing/pull/416

### CODE REVIEW

File: sale_fixed_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+To use this module, you need to:
+
+#. Go to *Sales*.
+#. Create a Sales Order and specify the type of discount and fixed/percent
+   discount in a line.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/10.0
```
Comment (nikul-serpentcs) [inline sale_fixed_discount/README.rst]: @jbeficent IMO Change Target `11.0`

File: sale_fixed_discount/views/account_invoice_views.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="account_invoice_line_form">
+            <field name="name">account.invoice.line.form</field>
+            <field name="model">account.invoice.line</field>
+            <field name="inherit_id" ref="account_invoice_fixed_discount.view_invoice_line_form"/>
+            <field name="arch" type="xml">
+                <data>
+                    <xpath expr="//field[@name='discount_fixed']" position="attributes">
```
Comment (nikul-serpentcs) [inline sale_fixed_discount/views/account_invoice_views.xml]: IMO here direct use `<field name="discount_fixed" position="attributes">`

File: sale_fixed_discount/views/account_invoice_views.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="account_invoice_supplier_form">
+            <field name="name">account.invoice.supplier.form</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account_invoice_fixed_discount.account_invoice_supplier_form"/>
+            <field name="arch" type="xml">
+                <data>
+                    <xpath expr="//field[@name='discount_fixed']" position="attributes">
```
Comment (nikul-serpentcs) [inline sale_fixed_discount/views/account_invoice_views.xml]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Changes

File: sale_fixed_discount/README.rst (hunk 1 above)
Comment (JordiBForgeFlow) [reply sale_fixed_discount/README.rst]: Thanks. I have attended to all your comments. The README will be automatically created.

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and technical review :+1:

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by lreficent** [comment]
@nikul-serpentcs please update your review.

**Comment by LoisRForgeFlow** [comment]
Ready to merge!

**Comment by LoisRForgeFlow** [comment]
Rebased to fix conflicts.

**Pull request merged**